src/spell.js

```diff
@@ -273,9 +274,11 @@ function parseObjectConditions(conditions) {
       });
     }
     else if (isLogicalCondition(value)) {
-      const condition = parseObjectValue(value);
-      normalizeLogicalCondition(condition, name);
-      result.push(condition);
+      const conds = parseObjectValue(value);
```

Comment:
maybe we can use `parseOperatorCondition(name, value)` here

Reply:
tried a bit, with no luck. I'd need more time to investigate on this.

Reply:
`parseOperatorCondition` seems can't parse logical operator and return object not object array